gwcs/geometry.py

```diff
@@ -195,8 +198,9 @@ def evaluate(self, x, y, z):
 
         h = np.hypot(x, y)
         lon = np.rad2deg(np.arctan2(y, x))
-        if h == 0.0:
-            lon *= 0.0
+
+        if isinstance(h, np.ndarray):
```

Comment:
Is it necessary to check `isinstance` here?  The model seems to always convert to array by this point anyway.

On the other hand, if it's possible to get to this point with primitive `h` and `lon`, we should probably multiply by 0.0 in that case as well, right?

Reply:
You are right. That was an appendix from a previous implementation of the fix (until I realized it is always an array).